<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>YAWYSIWYGEE</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionLoad"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionPrint"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuDebug">
    <property name="title">
     <string>Debug</string>
    </property>
    <addaction name="actionLoad_Test_txt"/>
    <addaction name="actionSave_As_Test_txt"/>
    <addaction name="actionTest_for_Memory_Leaks"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <widget class="QMenu" name="menuTheme">
     <property name="title">
      <string>Theme</string>
     </property>
     <addaction name="actionVanilla"/>
     <addaction name="actionChalkboard"/>
    </widget>
    <addaction name="actionZoom_In"/>
    <addaction name="actionZoom_Out"/>
    <addaction name="actionReset_Zoom"/>
    <addaction name="separator"/>
    <addaction name="actionShow_Line_Numbers"/>
    <addaction name="menuTheme"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionCopy_as_PNG"/>
    <addaction name="actionPaste"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuDebug"/>
   <addaction name="menuWindow"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="font">
    <font>
     <stylestrategy>PreferDefault</stylestrategy>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionFraction"/>
   <addaction name="actionMatrix"/>
   <addaction name="actionCases"/>
   <addaction name="actionBinom"/>
   <addaction name="actionLim"/>
   <addaction name="actionRoot"/>
   <addaction name="actionBigsum"/>
   <addaction name="actionBigint"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionSave_DOT_File">
   <property name="text">
    <string>Save DOT File</string>
   </property>
  </action>
  <action name="actionSave_As_Test_txt">
   <property name="text">
    <string>Save As Test File</string>
   </property>
  </action>
  <action name="actionLoad_Test_txt">
   <property name="text">
    <string>Load Test File</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="text">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="text">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionReset_Zoom">
   <property name="text">
    <string>Reset Zoom</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionShow_Line_Numbers">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Line Numbers</string>
   </property>
  </action>
  <action name="actionVanilla">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Vanilla</string>
   </property>
  </action>
  <action name="actionChalkboard">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Chalkboard</string>
   </property>
  </action>
  <action name="actionTest_for_Memory_Leaks">
   <property name="text">
    <string>Infinite Load/Delete Loop</string>
   </property>
   <property name="iconText">
    <string>Test for Memory Leaks
(Starts Infinite Loop)</string>
   </property>
   <property name="toolTip">
    <string>Infinite Load/Delete Loop</string>
   </property>
  </action>
  <action name="actionCopy_as_PNG">
   <property name="text">
    <string>Copy as PNG</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionFraction">
   <property name="icon">
    <iconset resource="qtypesetobjectresource.qrc">
     <normaloff>:/Toolbar/frac.png</normaloff>:/Toolbar/frac.png</iconset>
   </property>
   <property name="text">
    <string>Frac</string>
   </property>
   <property name="toolTip">
    <string>\frac</string>
   </property>
  </action>
  <action name="actionMatrix">
   <property name="icon">
    <iconset resource="qtypesetobjectresource.qrc">
     <normaloff>:/Toolbar/mat.png</normaloff>:/Toolbar/mat.png</iconset>
   </property>
   <property name="text">
    <string>Mat</string>
   </property>
   <property name="toolTip">
    <string>\mat</string>
   </property>
  </action>
  <action name="actionCases">
   <property name="icon">
    <iconset resource="qtypesetobjectresource.qrc">
     <normaloff>:/Toolbar/cases.png</normaloff>:/Toolbar/cases.png</iconset>
   </property>
   <property name="text">
    <string>Cases</string>
   </property>
   <property name="toolTip">
    <string>\cases</string>
   </property>
  </action>
  <action name="actionBinom">
   <property name="icon">
    <iconset resource="qtypesetobjectresource.qrc">
     <normaloff>:/Toolbar/binom.png</normaloff>:/Toolbar/binom.png</iconset>
   </property>
   <property name="text">
    <string>Binom</string>
   </property>
   <property name="toolTip">
    <string>\binom</string>
   </property>
  </action>
  <action name="actionLim">
   <property name="icon">
    <iconset resource="qtypesetobjectresource.qrc">
     <normaloff>:/Toolbar/lim.png</normaloff>:/Toolbar/lim.png</iconset>
   </property>
   <property name="text">
    <string>Lim</string>
   </property>
   <property name="toolTip">
    <string>\lim</string>
   </property>
  </action>
  <action name="actionRoot">
   <property name="icon">
    <iconset resource="qtypesetobjectresource.qrc">
     <normaloff>:/Toolbar/root.png</normaloff>:/Toolbar/root.png</iconset>
   </property>
   <property name="text">
    <string>Root</string>
   </property>
   <property name="toolTip">
    <string>\root</string>
   </property>
  </action>
  <action name="actionBigint">
   <property name="icon">
    <iconset resource="qtypesetobjectresource.qrc">
     <normaloff>:/Toolbar/bigint.png</normaloff>:/Toolbar/bigint.png</iconset>
   </property>
   <property name="text">
    <string>Int</string>
   </property>
   <property name="toolTip">
    <string>\bigint</string>
   </property>
  </action>
  <action name="actionBigsum">
   <property name="icon">
    <iconset resource="qtypesetobjectresource.qrc">
     <normaloff>:/Toolbar/bigsum.png</normaloff>:/Toolbar/bigsum.png</iconset>
   </property>
   <property name="text">
    <string>Sum</string>
   </property>
   <property name="toolTip">
    <string>\bigsum</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qtypesetobjectresource.qrc"/>
 </resources>
 <connections/>
</ui>
